src/components/Chart/model.ts

```diff
@@ -66,14 +66,25 @@ export interface PieChartDataPoint {
   color?: string
 }
 
+interface RangeAreaMask<XDomain> {
+  show?: boolean
+  showDots?: boolean
+  showDotsOffset?: XDomain
+  fillOpacity?: number
+  yAtEnd?: any
+  overDots?: boolean
```

Comment:
these two also, what do they mean?

Reply:
It means that the range series will overlay dots and lines on the chart. And I need a custom value to the chart line at the end of a range series (without its label)

Reply:
whats the difference between "hide the dots" and "overlay the dots"? And I still can't understand what "y at end" means 😅 can you exemplify with an image?